src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -109,8 +109,8 @@
  * input type and exceptions thrown are slightly different.
  *
  * <p><b><a id="sm-allowed">These methods behave differently depending on
```

Comment:
```suggestion
 * <p id="sm-allowed"><b>These methods behave differently depending on
```
We seem to mix `<p id` and `<p><a id` in our code, so both forms should work fine, and the `<p id` form seems to be a better fit here.

Reply:
Fixed it, thanks. (I tried not to touch code beyond what was necessary for this PR)